scripts/tests/test_zkai_d128_gate_value_compact_preprocessed_gate.py

```
@@ -0,0 +1,269 @@
+        payload = build_payload(summary, include_mutations=False)
+        for name in gate.MUTATION_NAMES:
+            mutated = gate.mutate_payload(payload, name)
+            with self.assertRaises(gate.GateValueCompactGateError, msg=name):
+                gate.validate_payload(mutated, summary, allow_missing_mutation_summary=True)
+
+    def test_rejects_record_item_count_drift(self):
+        summary = cli_summary()
+        record = summary["rows"][0]["local_binary_accounting"]["records"][1]
+        record["item_count"] -= 1
+        record["total_bytes"] = record["item_count"] * record["item_size_bytes"]
+        with self.assertRaisesRegex(gate.GateValueCompactGateError, "record counts drift"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)